### ISSUE TITLE
Add setting for overriding default "normal" cursor shape (eg. to blinking block etc)

### ISSUE THREAD

**Issue by damnskippy** [wincent/terminus#27]
Minor suggestion: would be nice to have a knob for user specified "normal" cursor shape

I'm probably in the minority but I set my normal cursor shape to blinking block (rather than a solid block) so it's easier to spot in a soup of terminal characters --  '0' instead of '2';  using urxvt (TERM=screen) running tmux.  At some point along the way I noticed the cursor stopped blinking and tracked down to terminus, which when going out of insert/restore modes, sets the cursor to normal shape which is set to solid block by default.  Perhaps an option to specify the initial cursor mode would be nice to have.

Just a minor suggestion, and thanks for your plugin(s).

**Comment by damnskippy** [comment]
Just to provide some more context, I was referring to this code here:
```
--- a/plugin/terminus.vim
+++ b/plugin/terminus.vim
@@ -48,7 +48,9 @@ if s:shape
     let s:start_replace="\<Esc>]50;CursorShape=" . s:replace_shape . "\x7"
     let s:end_insert="\<Esc>]50;CursorShape=" . s:normal_shape . "\x7"
   else
-    let s:cursor_shape_to_vte_shape={1: 6, 2: 4, 0: 2}
+    let s:cursor_shape_to_vte_shape={1: 6, 2: 4, 0: 0}
```

**Comment by wincent** [comment]
Yeah, that sounds like a totally reasonable thing to make customizable. Feel free to submit a PR if you'd like to scratch that itch; otherwise I will get to it at some point.

**Comment by ghost** [comment]
I would very much appreciate the same thing. 
Have you already addressed this issue? If not, how far is the progress? 
Thank you for the nice plugin!
